app/src/components/governance/ProposalHeader.vue

```diff
@@ -49,7 +49,7 @@
       <div class="proposer-and-summary-container">
         <div class="proposer">
           Proposed By:
-          {{ proposal.proposer.address | formatAddress }}
+          {{ proposal.address | formatAddress }}
```

Comment:
????

Reply:
This looks quite wrong

Reply:
i think fabo changed this in one of his PRs yesterday - but we should confirm of course

Reply:
no, doesn't look like. Schema is the same than before

Reply:
sorry might have been a mistake yesterday